### PR TITLE
[plugin.video.media-ccc-de@matrix] v0.2.0+matrix.1

### PR DESCRIPTION
**Pull request by TobiX**
### Description

This is the same as #3275, just for matrix. The changes are [minimal](https://github.com/voc/plugin.video.media-ccc-de/compare/matrix), since I have been writing PY2/3 compatible code for years (and there isn't much string handling in the add-on anyways)

### Checklist:
- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practice but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-plugins/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.
- If you see no activity on your PR after a week (so at least one weekend has passed) then please go to the #kodi-dev freenode IRC channel to reach out to the team

### CODE REVIEW

File: plugin.video.media-ccc-de/addon.xml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    <summary lang="de_DE">Videos von Chaos Computer Club Veranstaltungen.</summary>
+    <description lang="en_GB">This addon provides access to videos published on https://media.ccc.de/ (mostly lecture recordings of CCC events) </description>
+    <description lang="de_DE">Dieses Add-On ermöglicht den Zugriff auf Videos, die auf https://media.ccc.de/ veröffentlicht wurden (größtenteils Vortragsmitschnitte von CCC-Veranstaltungen)</description>
+    <news>v0.2.0 (2020-12-23)
+- The "F*** 2020" release
+- Remove option to fallback to unencrypted streams
+- Inital support for MPEG-DASH streams (#32)</news>
+    <language>de en</language>
+    <platform>all</platform>
+    <license>MIT License</license>
```
Comment (enen92) [inline plugin.video.media-ccc-de/addon.xml]: MIT only please (spdx identifier)

File: plugin.video.media-ccc-de/resources/lib/recording.py [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+from .helpers import user_preference_sorter, maybe_json
+
+
+class Recordings(object):
+    def __init__(self, json):
+        self.recordings = [Recording(elem) for elem in json['recordings']]
+
+    def recordings_sorted(self, quality, format, video=True):
+        print("Requested quality %s and format %s" % (quality, format))
```
Comment (enen92) [inline plugin.video.media-ccc-de/resources/lib/recording.py]: Drop the print and use xbmc.log only

File: plugin.video.media-ccc-de/resources/lib/stream.py [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+            if len(stream["urls"]) > 0:
+                for urlname, urldata in stream["urls"].items():
+                    self.streams.append(Stream(urlname, urldata, stream))
+        self.slug = json["slug"]
+        self.display = json["display"]
+        if len(group) > 0:
+            self.display = group + ": " + self.display
+
+    def streams_sorted(self, quality, format, dash=False, video=True):
+        print("Requested quality %s and format %s" % (quality, format))
```
Comment (enen92) [inline plugin.video.media-ccc-de/resources/lib/stream.py]: Same

**Review by enen92 — CHANGES_REQUESTED** [review]
Minimal changes needed. Thanks for submitting to matrix.
We wish you a merry Christmas ⛄

**Comment by TobiX** [comment]
Put all logging calls into a simple wrapper (so my tests still run 😁), so the calls should now be correct when running inside Kodi

**Review by enen92 — APPROVED** [review]

**Pull request merged**